app/views/albums/edit.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class="albums__topNav">
+  <%= button_to 'Home', root_path, method: 'get', class: "albums__homeButton" %>
+</div>
+
+<% if @album.errors.messages.present? %>
+  <%= render partial: "errors", locals: { record: @album } %>
+<% end %>
+
+<%= form_for @album do |f| %>
+  <table class="albums__table">
+    <col style="width:50%">
+    <col style="width:50%">
+    <tr class="albums__tableHeader">
+      <th colspan=2>
+        <h2>Edit Details</h2>
+      </th>
+    </tr>
+    <tr class="albums__tableRow">
+      <th>
+        <%= f.label :title %>
+      </th>
+      <th>
+        <%= f.text_field :title, value: @album.title, size: 50 %>
+      </th>
+    </tr>
+    <tr class="albums__tableRowAlt">
+      <th>
+        <%= f.label :artist %>
+      </th>
+      <th>
+        <%= f.text_field :artist, value: @album.artist, size: 50 %>
+      </th>
+    </tr>
+    <tr class="albums__tableRow">
+      <th>
+        <%= f.label :genre %>
+      </th>
+      <th>
+        <%= f.text_field :genre, value: @album.genre, size: 50 %>
+      </th>
+    </tr>
+    <tr class="albums__tableRowAlt">
+      <td>
+        all fields required
+      </td>
+      <td>
+        <%= f.submit 'Submit', data: { confirm: 'Are you sure?' } %>
```

Comment:
Is it necessary to confirm that I want to update my editing? It's not destructive, so if I make a mistake, I can just edit it again.